### PR TITLE
ES Module support

### PR DESCRIPTION
**Pull request by huan**
Link to:

- https://github.com/Chatie/tsconfig/issues/16

Issues:

1. There's no `ChatOpera` export from `@chatopera/sdk`

> Error: src/asker.ts(14,3): error TS2305: Module '"@chatopera/sdk"' has no exported member 'Chatopera'.

See:

https://github.com/wechaty/wechaty-chatopera/blob/45fb0a43e0fbf4e5d19c2cf2c8db19de7f55ada8/src/asker.ts#L12-L15

### CODE REVIEW

**Comment by huan** [comment]
@hailiang-wang Please help this PR to fix the `ChatOpera` import issue, then I will be able to continue working on it.

**Comment by huan** [comment]
ping @hailiang-wang

**Comment by huan** [comment]
Because the https://github.com/chatopera/chatopera-nodejs-sdk has been disabled the issue function, so I leave the reason here:

After inspecting the source code of `@chatopera/sdk`, I found that the typing was missing in the [index.d.ts](https://github.com/chatopera/chatopera-nodejs-sdk/blob/master/index.d.ts) file: there's no **Chatopera** definition at all.

I hope it can be resolved because it is breaking the importing code in the downstream TS programs.

**Comment by huan** [comment]
@mukaiu Could you please help us to review this PR, and if it's OK let's move on to merge it?

Thank you very much!

File: src/asker.ts [hunk 1]
```diff
@@ -3,11 +3,11 @@ import {
...
 
-import {
+import type {
   RepoConfig,
   ChatoperaOptions,
   ChatoperaResponse,
-} from './chatopera'
+}                       from './chatopera.js'
 
 import {
```
Comment (mukaiu) [inline src/asker.ts:12]: New code, need to add file extension.

**Review by mukaiu — COMMENTED** [review]
Merge the latest code to resolve eslint.

**Pull request merged**